### ISSUE TITLE
Panic on non-latin symbol

### ISSUE THREAD

**Issue by sikmir** [EdJoPaTo/mqttui#174]
```sh
$ RUST_BACKTRACE=1 mqttui
thread 'main' panicked at /cargo/registry/src/index.crates.io-6f17d22bba15001f/ratatui-0.26.2/src/text/line.rs:477:59:
byte index 67 is not a char boundary; it is inside 'ч' (bytes 66..68) of `{"order":1,"readonly":false,"title":{"en":"Enabled","ru":"Включен"},"type":"switch"}`
stack backtrace:
   0: rust_begin_unwind
             at /rustc/25ef9e3d85d934b27d9dada2f9dd52b1dc63bb04/library/std/src/panicking.rs:647:5
   1: core::panicking::panic_fmt
             at /rustc/25ef9e3d85d934b27d9dada2f9dd52b1dc63bb04/library/core/src/panicking.rs:72:14
   2: core::str::slice_error_fail_rt
   3: core::str::slice_error_fail
             at /rustc/25ef9e3d85d934b27d9dada2f9dd52b1dc63bb04/library/core/src/str/mod.rs:91:9
   4: <ratatui::text::line::Line as ratatui::widgets::WidgetRef>::render_ref
   5: <ratatui::text::text::Text as ratatui::widgets::WidgetRef>::render_ref
   6: ratatui::widgets::table::cell::Cell::render
   7: <ratatui::widgets::table::table::Table as ratatui::widgets::StatefulWidget>::render
   8: mqttui::interactive::details::Details::draw
   9: mqttui::interactive::App::draw
  10: mqttui::interactive::main_loop
  11: mqttui::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
$ mosquitto_sub -v -t /devices/buzzer/controls/enabled/meta
/devices/buzzer/controls/enabled/meta {"order":1,"readonly":false,"title":{"en":"Enabled","ru":"Включен"},"type":"switch"}
$ mqttui --version
mqttui 0.21.0
```

**Comment by EdJoPaTo** [comment]
Was fixed by https://github.com/ratatui-org/ratatui/pull/1089 and released in ratatui 0.27.3. I'll do a mqttui bugfix release including the fix for this.

**Comment by EdJoPaTo** [comment]
Released [v0.21.1](https://github.com/EdJoPaTo/mqttui/releases/tag/v0.21.1) just now. Thanks for reporting! :)
